scripts/test.rollup.config.js

```diff
@@ -5,6 +5,9 @@ import resolve from 'rollup-plugin-node-resolve';
 import worker from '@gkatsev/rollup-plugin-bundle-worker';
 import multiEntry from "rollup-plugin-multi-entry";
 
+/* to prevent going into a screen during rollup */
+process.stderr.isTTY = false;
```

Comment:
Ideally, we'll update rollup so we can set the clear screen option to false, but shouldn't hold up this PR.

Reply:
right after tba/lhls I planned on updating this project to generator conventions, where possible.